cardano-node/src/Cardano/Node/Run.hs

```diff
@@ -388,7 +389,8 @@ createDiffusionArguments publicIPv4SocketsOrAddrs
     -- merged into `ouroboros-networ`.
     { daIPv4Address = eitherSocketOrSocketInfo <$> publicIPv4SocketsOrAddrs
     , daIPv6Address = eitherSocketOrSocketInfo <$> publicIPv6SocketsOrAddrs
-    , daLocalAddress = fmap unSocketPath
+    , daLocalAddress = Just
```

Comment:
Was this expected?

Reply:
It probably wants a TODO to allow expressing the Nothing case in the config.

Reply:
I added a TODO comment